### DISCUSSION TITLE
Feedback: pasta cooking recipe OP

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#702] [Ideas]
Pasta can be cut from dough (crafted with 3 wheat and 1 egg) and yields 1 each, or: 1 per 1 wheat and 1/3 of an egg.

But, if cooked in the pot, it can be made from 2 wheat 1 egg, also skipping the cutting step which means no knife durability is used, AND it yields 4, or: 1 per 1/2 wheat and 1/4 of an egg (while again, not using knife durability?).

Was this intentional? I guess this requires you to have a cooking pot (crafted with iron) whereas a cutting board can be made out of wood, but without a cooking pot, pasta cannot be used yet anyways.

I suggest that it should only make 2 instead of 4. meaning you still _can_ make it directly from wheat and skip the cutting step, but would be slightly more cost effective to cut it manually.

**Comment by vectorwing** [comment]
The only way to craft Raw Pasta in "vanilla FD" is through the Cutting Board (wheat dough + knife).

From your description, I assume you're playing Farmer's Delight on a modpack. Keep in mind that modpacks will often include add-ons and custom datapacks, which might be tweaking the experience in various ways which weren't originally intended. That cooking recipe seems to be added by someone else.

Also, if you'd like to talk in more detail, feel free to join our Discord server; we got a community there to brainstorm ideas and ask questions too: https://discord.gg/eFsz5SK

**Reply by Aceplante** [reply]
As I mentioned in other replies, it was likely due to one of the other branch mods such as cultural delights. I will try to pay attention to if it is your mod of his before I report issues in the future
